docs/source/test_set.rst

```diff
@@ -34,4 +35,23 @@ To run the test set on a pre-trained model, use this method.
     trainer.test(model)
 
 In this  case, the options you pass to trainer will be used when
-running the test set (ie: 16-bit, dp, ddp, etc...)
\ No newline at end of file
+running the test set (ie: 16-bit, dp, ddp, etc...)
+
+
+Test additional data loaders
+----------------------------
+If no test sets were passed in to `trainer.fit()` or otherwise declared inside your
```

Comment:
test sets can't be passed into trainer.fit
also, if they are declared in LightningModule, they get ignored and only the new dataloader(s) get used. I'm 99% sure.

Reply:
https://github.com/PyTorchLightning/pytorch-lightning/blob/8ee6d91d0e8ea83829a04d35e7fe7500eb332211/pytorch_lightning/trainer/trainer.py#L779-L784

Reply:
@awaelchli is right here...

Reply:
where? I only see val dataloaders

Reply:
This is my mistake. In my brain, I still had the code base for old version of PTL when test_dataloaders was indeed an argument passed into `.fit`: [see here](https://github.com/PyTorchLightning/pytorch-lightning/blob/ee68d5ba8e66bba52b3cc3d9d8780d6df2f3204b/pytorch_lightning/trainer/trainer.py#L602). But toady that is no longer the case.

Reply:
I say that @awaelchli is correct, test set cannot be passed to `.fit()`

Reply:
A slight reformulation would do it I think.